CodeStarterCamp_Week2/Week2Step3.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Week2Step3.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 홍세희 on 2023/08/29.
+//
+
+var allLottoNumberStorage: Dictionary<String , Set<Int>> = [String : Set<Int>]()
```

Comment:
Dictionary 타입을 잘 설정해주셨네요! 👍

다만, [애플 공식문서](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/collectiontypes/#Dictionary-Type-Shorthand-Syntax)에 따르면 Dictionary의 타입을 나타낼 때 축약형 표현을 선호한다고 합니다.

좀 더 간략하고 보기 쉽기 때문인 것 같네요 🤔
short form을 활용하면 아래와 같이 간단히 표현할 수 있습니다.
Dictionary 타입을 표현할 때 보통 colon의 뒤에만 공백을 한 칸 줍니다.

```suggestion
var lottoNumberStorage: [String: Set<Int>] = [:]
```

---

추가로, step2 에서는 숫자의 순서가 상관이 없었지만
일련의 숫자를 저장할 땐 보통 `정렬`을 하죠!

* Dictionary의 value 타입을 순서가 있는 콜렉션 타입으로 변경해주세요!
* Set의 인스턴스 메서드 중에서 내부 항목들을 정렬(`sort`)하는 메서드를 찾아봅시다. 💪